download-templates.sh

```diff
@@ -20,5 +20,10 @@ for item in ${version}; do
     curl -f -o "${KUBEVIRT_TEMPLATE_DIR}/${item}/vm-template-fedora.yaml" "${vm_template_url}/${item}/cluster/vm-template-fedora.yaml"
     curl -f -o "${KUBEVIRT_TEMPLATE_DIR}/${item}/vm-template-rhel7.yaml" "${vm_template_url}/${item}/cluster/vm-template-rhel7.yaml"
     curl -f -o "${KUBEVIRT_TEMPLATE_DIR}/${item}/vm-template-windows2012r2.yaml" "${vm_template_url}/${item}/cluster/vm-template-windows2012r2.yaml"
+    wget -O "${KUBEVIRT_TEMPLATE_DIR}/${item}/demo-content.yaml" "${manifest_url}/${item}/demo-content.yaml"
+    if [ ! -s "${KUBEVIRT_TEMPLATE_DIR}/${item}/demo-content.yaml" ];
+    then
+        rm -f "${KUBEVIRT_TEMPLATE_DIR}/${item}/demo-content.yaml"
```

Comment:
I'm not sure what this is doing.  What's this checking for and why would you remove the file?

Reply:
In older releases (before v0.3.0) demo-content.yaml didn't exist, and in this case wget fails with 404 and leave empty file demo-content.yaml, so in case demo-content is empty I just remove it.

Reply:
You can do ```curl -f -o "${KUBEVIRT_TEMPLATE_DIR}/${item}/demo-content.yaml"``` instead and it won't drop and empty file in there.  It will just fail.